packages/api/tests/routes.test.js

```diff
@@ -1,9 +1,34 @@
-const request = require('supertest')
-const app = require('../server')
-describe('GET Endpoints', () => {
-  it('should create a new GET request', async () => {
+const request = require('supertest');
+const app = require('../server');
+const db = require('../models');
+
+describe('GET Nomination Endpoint', () => {
+  let nomination;
+
+  beforeAll(() => {
+    nomination = db.Nomination.build({ id: 'b5a27641-d76b-4ab0-9cef-8bf9eb9e8fab' });
+    return nomination.save();
+  });
+
+  afterAll(() => {
+    nomination.destroy();
+  });
+
+  it('when nomination exist it should return a 200', async () => {
     const res = await request(app)
-      .get('/greeting')
-    expect(res.statusCode).toBe(200)
-  })
-})
+      .get(`/nomination/${nomination.id}`);
+    expect(res.statusCode).toBe(200);
+  });
+
+  it('when nomination does not exist it should return a 404', async () => {
+    const res = await request(app)
+      .get('/nomination/00000000-0000-0000-0000-000000000000');
+    expect(res.statusCode).toBe(404);
+  });
+
+  it('when id is not a valid uuid it should return a 404', async () => {
```

Comment:
Per my earlier feedback, this test should be for a 400 code.

(Also I have the same language nitpick:

`'returns a 400 when provided id is not a valid uuid'`